readout.sh

```diff
@@ -32,7 +32,12 @@ esac
 
 # Enforce no warning code in the PR checker
 if [[ $ALIBUILD_O2_TESTS ]]; then
-  CXXFLAGS="${CXXFLAGS} -Werror -Wno-error=deprecated-declarations"
+  if [[ $ARCHITECTURE != osx ]] ; then
```

Comment:
Did you actually test it? I think this will not work.

```
case $ARCHITECTURE in 
  osx*) ;;
  *) CXXFLAGS="${CXXFLAGS} -Werror -Wno-error=deprecated-declarations" ;;
esac
```

is probably what you want (which has also the advantage of being posix compliant).

Reply:
Oups, just on the linux commandline thb. That's what you get for being sloppy I guess 😅

Reply:
To be clear, the issue is that the architecture name is `osx_x86-64`, not `osx`.